.github/workflows/ci.yml

```diff
@@ -65,6 +55,60 @@ jobs:
           (cd pyvortex && uv run maturin develop)
 
           (cd docs && uv run make html)
+
+  rust-docs:
```

Comment:
weirdly the command pyvortex runs also does this

Reply:
yeah `make -C docs html` builds _all_ the docs, including the Rust docs. We could add a different target like `make -C docs full-html` and use that in docs.yml.